CWE/history/countries/SYR - Syria.txt

```diff
@@ -2,7 +2,7 @@ capital = 902
 primary_culture = arab_levantine
 culture = arab
 religion = secularism
-government = mandate_government
+government = democracy
```

Comment:
Syria should start as a mandate in 1946. They only became a republic on independence.

Reply:
Alright, should the flag for Syria a as a Mandatory Government be changed to the green-white-black one, because from what I can find, that was the official flag of Syria as a Mandate by 1946?

Reply:
Alright, the flag can be changed.